zetaclient/bitcoin_client.go

```diff
@@ -815,12 +815,13 @@ func (ob *BitcoinChainClient) findNonceMarkUTXO(nonce uint64, txid string) (int,
 //
 // Parameters:
 //   - amount: The desired minimum total value of the selected UTXOs.
-//   - utxoCap: The maximum number of UTXOs to be selected.
+//   - utxos2Spend: The maximum number of UTXOs to spend.
 //   - nonce: The nonce of the outbound transaction.
+//   - csldtRank: The rank below which UTXOs will be consolidated.
 //   - test: true for unit test only.
 //
 // Returns: a sublist (includes previous nonce-mark) of UTXOs or an error if the qulifying sublist cannot be found.
```

Comment:
I think we can update this comment to describe the new values returned in the function